apps/api/prisma/schema.prisma

```diff
@@ -343,12 +349,14 @@ model DocumentVersion {
   mime                   String?
   horizonDataID          String?
   fileMD5                String?
+  documentId             String?
```

Comment:
@LetsMakeItTechnical Would be good to come to an agreement regarding what we mean by `id`, `documentId`, `horizonDataId` with @DavidBeale @Philip-Philippou1

Reply:
documentId and version will need to be a primary key

Reply:
Seems like this has had one rebase too many - so I will clean it up and push again